### ISSUE TITLE
[BUG] Sample feature output visualisation is incorrect

### ISSUE THREAD

**Issue by allenphilip93** [opendistro-for-elasticsearch/anomaly-detection#407]
**Description:**
The sample feature output visualisation is shown differently from the actual value

Actuals:
![image](https://user-images.githubusercontent.com/20537002/159632316-ed200000-dae7-46af-a4b5-b09649e27583.png)

Sample Feature Output:
![image](https://user-images.githubusercontent.com/20537002/159632550-df8ac68f-e7e1-4e0c-a394-636fa79ebb28.png)

I've taken the sample index and applied the same filters in both the cases. For the feature I've aggregated by count on one of the filtered features.
![image](https://user-images.githubusercontent.com/20537002/159632827-d1aa08f0-c854-4ce1-a7ec-54e88f82d2a1.png)

ES Version: 1.10.1

**Comment by kaituo** [comment]
The actual graph showed it is per 12 hours. Does it mean you have a count of doc.transactionEvent.store_name.keyword every 12 hours? What intervals do you use for the detector?

**Comment by allenphilip93** [comment]
Yes, I've configured the anomaly detector interval for 12h with default window size (8 intervals). I'm also using the same field for x-axis (lastModifiedOn) for both the cases.

**Comment by kaituo** [comment]
Tried to reproduce your bug without success. It is hard to make the graph look exactly the same as AD dashboard and Kibana Visualization use different start and end timestamps to aggregate data. For example, even though both graphs use 5 minute interval, AD dashboard uses the range between 1650180367561 and 1650180367561 to aggregate data, while Kibana Visualization uses the range between 1650180317561 and 1650180317561 to aggregate data. But the rough trend and shape of the line should match. But from your graphs, we have totally different things. Could you help test if this is related to interval? For example, what if you choose 5 minute interval? At the same time, I will start a 12 hour interval detector. But it will take time for 

Also, yes, the plot for Sample Feature Breakdown is essentially the input time series feature for the detector in the same default timezone.

Example 1 (5 min interval):
<img width="1612" alt="Screen Shot 2022-04-18 at 10 32 34 AM" src="https://user-images.githubusercontent.com/5303417/163848651-abb3606e-992b-4fae-8c67-82ce123ab662.png">

![Screen Shot 2022-04-18 at 10 32 43 AM](https://user-images.githubusercontent.com/5303417/163848675-2b1c5eb6-23ad-4f5c-87e9-0fc60a76cdce.png)

Example 2 (5 minute interval):
![Screen Shot 2022-04-18 at 10 17 21 AM](https://user-images.githubusercontent.com/5303417/163848751-fd82116c-457c-4a11-a650-f9f86bbed09a.png)

![Screen Shot 2022-04-18 at 10 17 06 AM](https://user-images.githubusercontent.com/5303417/163848814-1b575a60-8a52-43d4-90e2-90a7aead9685.png)

Example 3 (12 hr interval):
![Screen Shot 2022-04-18 at 12 29 31 PM](https://user-images.githubusercontent.com/5303417/163865253-cf310c60-04c0-41f5-b50c-59413311e373.png)

![Screen Shot 2022-04-18 at 12 29 40 PM](https://user-images.githubusercontent.com/5303417/163865267-027661fa-49f7-48d8-9321-4c029cda03ed.png)
